src/pages/ProductPage/ProdDetail/ProdDetail.scss

```
@@ -1,29 +1,29 @@
-.product-detail{
+.product-detail {
   font-weight: bold;
   margin-top: 60px;
 
-  &__title{
+  &__title {
     font-size: 35px;
     margin-top: 12px;
   }
   
-  &__subtitle{
+  &__subtitle {
     color: orange;
     font-size: 12px;
     margin-top: 15px;
     margin-bottom: 30px;
   }
 
-  &__description{
+  &__description {
     font-weight: 200;
     margin-top: 16px;
   }
 
-  &__price{
+  &__price {
     font-size: 25px;
     margin-top: 30px;
   
-    & span{
+    & span {

```

```suggestion
    span {
```

Same [here](https://github.com/MarcelloAragoni/ecommerce/pull/6/files/6fc30bf3e83fbff74c031268e5cfc36b292003ee..a1030f2bca6043093524a8991bbaf5309bf462f3#r862813392)